plugins/jobs/girder_plugin_jobs/__init__.py

```diff
@@ -43,6 +44,12 @@ def scheduleLocal(event):
         fn(job)
 
 
+@plugin.config(
+    name='jobs',
+    description='A general purpose plugin for managing offline jobs.',
+    url='http://girder.readthedocs.io/en/latest/plugins.html#jobs',
+    version='2.0.0'
+)
```

Comment:
This is the entry point decorator providing the information that used to come from `plugin.json`.